Sources/Components/RealestateSoldState/RealestateSoldStateView.swift

```diff
@@ -22,6 +22,28 @@ public class RealestateSoldStateView: UIView {
         return view
     }()
 
+    private lazy var userContactMethodView: UserContactInformationView = {
+        let view = UserContactInformationView(withAutoLayout: true)
+        view.configure(
+            with: "Hvordan kan vi kontakte deg?",
+            contactMethodModels: [
+                .init(
+                    identifier: "1",
```

Comment:
Will these strings be moved to the demo data or something? 😄 
the strings should be localized in ios-app

Reply:
Yup, that's changed in the later PRs😌 Should probably have noted that in the description😄